### PR TITLE
Adds regular leg actuators to mecha fabricators and tweaks their presence on hardsuits

### PR DESCRIPTION
**Pull request by Alberyk**
-gives a sprite to the leg actuators, mounted kinect gun and mounted termal drill
-adds regular actuators to the mecha fabricator, with enough tech and materials
-gives the ert leader the military version of the actuators, while the rest gets regular ones
-tweaks some modules around the hardsuits

### CODE REVIEW

File: code/modules/research/mechfab_designs.dm [hunk 1]
```diff
@@ -721,6 +721,14 @@
 	materials = list(DEFAULT_WALL_MATERIAL= 30000, "phoron" = 12500, "silver" = 10000, "gold" = 10000)
 	build_path = /obj/item/rig_module/device/rcd
 
+/datum/design/hardsuitmodules/actuators_module
+	name = "leg actuators"
+	desc = "A set of electromechanical actuators, for safe travesal of multilevelled areas."
```
Comment (LordFowl) [inline code/modules/research/mechfab_designs.dm]: Spelling. Traversal.

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 2]
```diff
@@ -478,10 +478,13 @@
 /obj/item/rig_module/actuators
 	name = "leg actuators"
 	desc = "A set of electromechanical actuators, for safe travesal of multilevelled areas."
-	icon_state = "generic"
+	icon_state = "motor"
```
Comment (LordFowl) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm]: Literally a codersprite.

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 3]
```diff
@@ -478,10 +478,13 @@
 /obj/item/rig_module/actuators
 	name = "leg actuators"
 	desc = "A set of electromechanical actuators, for safe travesal of multilevelled areas."
```
Comment (LordFowl) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm]: Spelling. Traversal.

File: code/modules/clothing/spacesuits/rig/modules/utility.dm (hunk 2 above)
Comment (Alberyk) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm]: Better than the old one that was the green cube.

**Comment by skull132** [comment]
Slight speedmeme due to test starting.

**Pull request merged**